### PR TITLE
Logging for input validation should be ZP-specific

### PR DESCRIPTION
**Pull request by j053ph4**
- Logging of input validation-related issues should follow each ZP's
load_yaml arguments

### CODE REVIEW

**Comment by j053ph4** [comment]
This is a relatively minor issue but could cause some unwanted logging

File: ZenPacks/zenoss/ZenPackLib/lib/base/types.py [hunk 1]
```diff
@@ -58,10 +60,10 @@ def validate(cls, value):
         # truncate or pad with 0s
         def check_length(value):
             if len(value) > 6:
-                LOG.warning('Max length exceeded, truncating: {}'.format(value))
+                cls.LOG.warning('Max length exceeded, truncating: {}'.format(value))
```
Comment (dbouchillon) [inline ZenPacks/zenoss/ZenPackLib/lib/base/types.py]: Should this be warning or debug?

File: ZenPacks/zenoss/ZenPackLib/lib/base/types.py [hunk 2]
```diff
@@ -58,10 +60,10 @@ def validate(cls, value):
...
                 value = value[:6]
             elif len(value) < 6:
-                LOG.warning('Min length not met, padding: {}'.format(value))
+                cls.LOG.warning('Min length not met, padding: {}'.format(value))
```
Comment (dbouchillon) [inline ZenPacks/zenoss/ZenPackLib/lib/base/types.py]: should this be warning or debug?

File: ZenPacks/zenoss/ZenPackLib/lib/base/types.py [hunk 3]
```diff
@@ -118,18 +121,18 @@ def validate(cls, value):
         try:
             value = int(value)
             if value < 0:
-                LOG.warning("Invalid severity value ({}), increasing to 0".format(value))
+                cls.LOG.warning("Invalid severity value ({}), increasing to 0".format(value))
```
Comment (dbouchillon) [inline ZenPacks/zenoss/ZenPackLib/lib/base/types.py]: warning or debug

File: ZenPacks/zenoss/ZenPackLib/lib/base/types.py [hunk 4]
```diff
@@ -118,18 +121,18 @@ def validate(cls, value):
...
                 value = 0
             elif value > 5:
-                LOG.warning("Invalid severity value ({}), reducing to 5".format(value))
+                cls.LOG.warning("Invalid severity value ({}), reducing to 5".format(value))
```
Comment (dbouchillon) [inline ZenPacks/zenoss/ZenPackLib/lib/base/types.py]: warning or debug

File: ZenPacks/zenoss/ZenPackLib/lib/base/types.py [hunk 5]
```diff
@@ -118,18 +121,18 @@ def validate(cls, value):
...
                 value = 5
         except (TypeError, ValueError):
             if isinstance(value, str):
                 if value.lower() in cls._valid_text:
                     value = cls.to_num.get(value.lower())
                 else:
                     sev_num_txt = [str(x) for x in cls._valid_num]
-                    LOG.warning("Invalid severity value ({}), "\
+                    cls.LOG.warning("Invalid severity value ({}), "\
```
Comment (dbouchillon) [inline ZenPacks/zenoss/ZenPackLib/lib/base/types.py]: warning or debug

**Pull request merged**